addons/stock/models/stock_inventory.py

```diff
@@ -374,28 +374,32 @@ def _run_inventory_tasks(self, company_id=False):
                                       'location_ids': location})
             self.create(location_vals)
         # conflict inventories handled in separate function so inventories can also be updated when user opens inventory menuitem
-        self._run_conflict_inventory_tasks(company_domain=company_domain)
+        self._run_conflict_inventory_tasks(company_id=company_id)
 
     @api.model
-    def _run_conflict_inventory_tasks(self, **kwargs):
+    def _run_conflict_inventory_tasks(self, company_id=False):
         """ Updates/creates/deletes conflict inventories. Conflict inventories include:
                 - negative quantities values, created per warehouse
             Note that an in progress inventory will prevent an update/unlink when its
             corresponding values are out of date.
         """
-        company_domain = kwargs.get('company_domain')
-        if not company_domain:
-            company_domain = [('company_id', 'in', self.env.user.company_ids.ids)]
+        company_domain = [('company_id', '!=', False)]
```

Comment:
If we do this, then when we click on the "Inventory Adjustments" menu, all company inventories will be recalculated rather than just the user's currently selected ones, won't they?

Reply:
Oh ok I get it, sorry I though you were using the JS `and` for the Python `and`

Reply:
@ticodoo Since we don't use sudo (via menu items). search will only pick the user company